### PR TITLE
Fix Docker Builds

### PR DESCRIPTION
**Pull request by wrathagom**
Added path to Docker commands. Temporarily added Tensorflow to basic requirements.

@tmbo NLU is failing to install for me, asking for Tensorflow, even when I am not using the tensor flow pipeline. I've temporarily added Tensorflow to the base requirements, but can merge in a real fix if you can point me to it.

**Proposed changes**:
- Add `, "--path", "/app/projects/"` to the Docker commands for spacy and full.

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by amn41** [comment]
@Ghostvv - can you please look into the tensorflow thing? looks like it's imported globally somewhere (possibly because of the registry)

**Comment by Ghostvv** [comment]
yes it is, in the new embedding classifier. After we introduced new persist scheme, tensor flow was imported globally. Shall I change back and import it in every file?

**Comment by Ghostvv** [comment]
https://github.com/RasaHQ/rasa_nlu/pull/959

**Comment by amn41** [comment]
thanks @Ghostvv ! we'll merge that - @wrathagom can you please remove tf from basic requirements again?

File: docker/Dockerfile_full [hunk 1]
```diff
@@ -94,4 +94,4 @@ COPY sample_configs/config_spacy_duckling.yml ${RASA_NLU_HOME}/config.yml
 EXPOSE 5000
 
 ENTRYPOINT ["./entrypoint.sh"]
-CMD ["start", "-c", "config.yml"]
+CMD ["start", "-c", "config.yml", "--path", "/app/"]
```
Comment (amn41) [inline docker/Dockerfile_full]: why are the paths different for the two dockerfiles?

Comment (wrathagom) [reply docker/Dockerfile_full]: shouldn't be. Originally I had it just at `/app/` before realizing before fully realizing what the path command was for. fixing.

**Pull request merged**